content/en/docs/reference/command-line-tools-reference/feature-gates/remove-self-link.md

```diff
@@ -1,4 +1,5 @@
 ---
+# Removed from Kubernetes
```

Comment:
```suggestion
removed: true
```

is what's actually needed here.

If you can, rebase against the upstream _dev-1.30_ branch too.